clang/lib/DPCT/MigrateScript/MigrateCmakeScript.cpp

```diff
@@ -20,6 +20,7 @@
 #include <algorithm>
 #include <cstring>
 #include <fstream>
+#include <numeric>
```

Comment:
Line 23 seem  not be used in this file, can it be removed?

Reply:
std::sccumulate required <numeric> inclusion